onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -203,6 +203,10 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
               "The value to be filled in the attention mask. Default value is -10000.0f",
               AttributeProto::FLOAT,
               OPTIONAL_VALUE)
+        .Attr("mup_scale",
```

Comment:
How about just use norm_factor here? In this way, we can get the value from Div in the graph directly.

Reply:
sure